src/Shared/BenchmarkRunner/DefaultCoreConfig.cs

```diff
@@ -29,6 +29,9 @@ public DefaultCoreConfig()
 
             AddValidator(JitOptimizationsValidator.FailOnError);
 
+            // So that the using statement, above, doesn't appear as a warning for being unused
+            _ = NetCoreAppSettings.NetCoreApp20;
```

Comment:
Weird, but fine since these are just benchmarks.

Reply:
This is interesting to me too, the usings are unused but we can't remove them?

Reply:
They're used, but only inside conditional `#if` blocks, so when looking in the IDE, unless you are looking in that context, they appear unused (and you get a warning for them in the error list). The alternative would have been to add the right `#if` around the using statement, but I thought thats a bit ugly. Happy to change if people disagree?

The roslyn issue is https://github.com/dotnet/roslyn/issues/49925